site/content/en/faq.md

```diff
@@ -388,46 +258,334 @@ information, or simply redirect the reader to your `LICENSES/` directory.
-licensing info applies to only a certain part of a file.
+## ... override erroneous REUSE information? {#override-info}
+
+You may have a file that contains copyright or licensing tags that are wrong.
+This may be on purpose, or maybe you have good reasons not to edit these files.
+You have a few options to instruct REUSE to ignore the erroneous information and
+to use correct information instead.
+
+1. Use [the `REUSE-IgnoreStart` and `REUSE-IgnoreEnd` tags](#exclude-lines)
+   inside of the file.
+2. Use [a `.license` file to override the contents of the
+   file](#uncommentable-file).
+3. Use [a `REUSE.toml` file](#bulk-license) with the `precedence = "override"`
+   key-value.
+
+<!-- REUSE-IgnoreStart -->
+
+## ... add additional REUSE information to a file without editing the header? {#aggregate-info}
+
+There is a specific corner case where a file may contain valid REUSE information
+(usually copyright holders), but it is not desirable for you to edit this file's
+comment header by hand, even though there is more information that should apply
+to this file. A good example of this corner case is `.po` files provided by
+translators. These may or may not contain copyright and licensing information,
+depending on the translator's translation software and settings.
+
+You can add additional information using the `precedence = "aggregate"`
+key-value in your [`REUSE.toml` file](#bulk-license). An example looks like
+this:
+
+```toml
+[[annotations]]
+path = "po/*.po"
+precedence = "aggregate"
+SPDX-FileCopyrightText = "Contributors to My Project"
+SPDX-License-Identifier = "GPL-3.0-or-later"
+```
+
+In the above example, 'Contributors to My Project' is added to the file's
+copyright holders (if any), and 'GPL-3.0-or-later' is added to the file's
+licenses (if any).
+
+## ... copy someone else's work? {#copy-work}
+
+If someone else has made their work available for you to use and copy, you can
+incorporate their work into your project. When you put the work in one of the
+files in your project, you should add an `SPDX-FileCopyrightText` tag for the copyright
+holder(s) and an `SPDX-License-Identifier` tag for the license(s) under which
+the work was made available.
+
+If the work was licensed differently from your project, you should [verify
+whether the licenses are compatible](#license-compatibility), and add the new
+licenses to your project.
+
+## ... copy a work that has no copyright notice or license? {#no-copyright-license}
+
+Before you proceed, always first make sure that you can find the copyright and
+licensing information elsewhere. Some projects only include this information in
+the root directory or in their README file.
+
+If you can find no copyright notice, you can try to add it yourself by divining
+the copyright holder from the author. See [this
+question](#copyright-holder-author) for more information on the distinction.
+When in doubt, contact the author for clarification.
```

Comment:
```suggestion
When in doubt, contact the author for clarification.

It may also be that the author specifically wishes to remain pseudonymous or completely anonymous, which is their right. In this case, write their pseudonym, or if they do not want even that, then write `NONE`.
```

Another option would be `NOASSERTION`, but I’m leaning towards `NONE`.

From https://spdx.github.io/spdx-spec/v2.3/file-information/#88-copyright-text-field:

> Identify the copyright holder of the file, as well as any dates present. This shall be a free-form text field extracted from the actual file.
> 
> The options to populate this field are limited to:
> 
> Any text relating to a copyright notice, even if not complete;
> 
> NONE, if the file contains no copyright information whatsoever; or
> 
> NOASSERTION, if
> 
>     the SPDX document creator has made no attempt to determine this field; or
> 
>     the SPDX document creator has intentionally provided no information (no meaning should be implied from the absence of an assertion).
> 
> If the Copyright Text field is not present for a file, it implies an equivalent meaning to NOASSERTION. The metadata for the copyright text field is shown in Table 43.

Reply:
Looks good, added to the FAQ!